features/support/apiREST.js

```diff
@@ -14,13 +14,11 @@ ApiREST.prototype.init = function (world) {
 ApiREST.prototype.disconnect = function () {};
 
 ApiREST.prototype.pathApi = function (path) {
-  var basePath = '/api';
-
-  if (apiVersion) {
-    basePath += '/' + apiVersion;
-  }
+  return config.url + '/api/1.0/' + path;
```

Comment:
hmm why "1.0" is hard-coded ?
Should it be rather "return config.url + '/api/' + apiVersion + path;" ?